spec/rspec/expectations/configuration_spec.rb

```diff
@@ -157,6 +157,24 @@ def delegated?; true; end
           configure_syntax(@orig_syntax)
         end
 
+        it "warns when should syntax is choosen" do
+          configure_syntax :expect
+
+          configuration_arguments = [
+            "Expectations syntax configuration",
+            {:call_site=>nil, :replacement=>"the default `expect` syntax"},
+          ]
+          expect(RSpec).to receive(:deprecate).with(*configuration_arguments).at_least(:once)
+
+          syntax_arguments = [
+            "`:should` Expectations syntax",
+            {:call_site=>nil, :replacement=>"the default `expect` syntax"},
+          ]
+          expect(RSpec).to receive(:deprecate).with(*syntax_arguments).at_least(:once)
```

Comment:
I need to add `.at_least(:once)` because of line 157 that leak into the test. If you comment it out, there is not need for `.at_least(:once)` anymore.